stravalib/client.py

```diff
@@ -1374,48 +1374,6 @@ def get_segment_streams(self, segment_id, types=None, resolution=None,
         # Pack streams into dictionary
         return {i.type: i for i in streams}
 
-    def get_running_race(self, race_id):
-        """
-        Gets a running race for a given identifier.
-
-        TODO: This has been deprecated by strava as of Nov 1 2021. See
-        https://developers.strava.com/docs/changelog/
-        https://developers.strava.com/docs/reference/#api-models-RunningRace
-
-        :param race_id: id for the race
-
-        :rtype: :class:`stravalib.model.RunningRace`
-        """
-        raw = self.protocol.get('/running_races/{id}', id=race_id)
-        return model.RunningRace.deserialize(raw, bind_client=self)
-
-
-    def get_running_races(self, year=None):
-        """
-        Gets a running races for a given year.
-
-        TODO: this has been deprecated by strava as of Nov 1 2021 - See
-        https://developers.strava.com/docs/changelog/
-        https://developers.strava.com/docs/reference/#api-RunningRaces-getRunningRaces
-
-        :param year: year for the races (default current)
-
-        :return: An iterator of :class:`stravalib.model.RunningRace` objects.
-        :rtype: :class:`BatchedResultsIterator`
-        """
-        if year is None:
-            year = datetime.datetime.now().year
-
-        params = {"year": year}
-
-        result_fetcher = functools.partial(self.protocol.get,
-                                           '/running_races',
-                                           **params)
-
-        return BatchedResultsIterator(entity=model.RunningRace, bind_client=self,
```

Comment:
can you add some explain why delete this in this PR or commit message

Reply:
These endpoints don't exist anymore at Strava, and the current build fails because of that.